app/src/main/java/org/mozilla/fenix/search/awesomebar/AwesomeBarView.kt

```diff
@@ -197,10 +197,12 @@ class AwesomeBarView(
 
         if (!searchEngine.isNullOrEmpty()) {
             searchEngine = when (searchEngine) {
-                GOOGLE_SEARCH_ENGINE_NAME -> activity.getString(
+                GOOGLE_SEARCH_ENGINE_NAME -> getString(
```

Comment:
This was the only way I found to mock in tests the response from the `activity.getString()` call to be able to test the header that we send.
Otherwise we would've need to:
- instantiate `HomeActivity` and it's a much too complex process to get past it's initialization 
- use `mockkStatic(Context::Class)` to be able to mock the response for that call but that didn't work either, was throwing errors.